src/BSD-3-Clause.xml

```diff
@@ -26,8 +26,9 @@
             </item>
             <item>
                 <bullet>3.</bullet>
-                Neither the name of <alt match=".+" name="organizationClause3">the copyright holder</alt> 
-                 nor the names of its contributors may be used to endorse or promote products derived from this 
+                <alt match="(The name of.+may not)|(Neither the name of.+nor the names of its contributors may)" name="organizationClause3">
```

Comment:
Maybe `.+` -> ` .+ ` (requiring wrapping whitespace) for both `.+`.

Reply:
Thought we would leave out the spaces in case there are other white space characters used (e.g. linefeed).

Reply:
> Thought we would leave out the spaces in case there are other white space characters used (e.g. linefeed).

But that applies to all template whitespace, no?  I think linefeed -> space normalization is preprocessing that tools need to do before comparing with the template.